Dnn.AdminExperience/EditBar/Dnn.EditBar.Library/Dnn.EditBar.Library.csproj

```diff
@@ -39,22 +39,13 @@
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\bin\Newtonsoft.Json.dll</HintPath>
+      <HintPath>..\..\..\DNN Platform\DotNetNuke.Web\bin\Newtonsoft.Json.dll</HintPath>
```

Comment:
Should this point to the NuGet `packages` folder?  This `bin` folder isn't part of the repo, it's created as a consequence of building; however, I think it would be best if the reference worked just by restoring NuGet packages, not also requiring a build.

Reply:
Do you mean to update the project Dnn.EditBar.Library using NuGet manager to update the newtonsoft.json rather than importing it from DNN Platform\DotNetNuke.Web\ project bin folder?

Reply:
Yes

Reply:
Fixed in Commit 0305e2c9